x-pack/plugins/maps/public/elasticsearch_geo_utils.js

```diff
@@ -400,6 +400,43 @@ export function getBoundingBoxGeometry(geometry) {
   return formatEnvelopeAsPolygon(extent);
 }
 
+export function makeESBbox({ maxLat, maxLon, minLat, minLon }) {
```

Comment:
This function is duplicate of `createGeoBoundBoxFilter`. Just use `createGeoBoundBoxFilter`